src/v2/view-builder/views/ov/SameDeviceEnrollLink.js

```diff
@@ -0,0 +1,51 @@
+import { View } from '@okta/courage';
+import hbs from '@okta/handlebars-inline-precompile';
+import { FORMS as RemediationForms } from '../../../ion/RemediationConstants';
+
+export default View.extend({
+  className: 'ov-same-device-enroll-text',
+  template: hbs `
+        {{{i18n code="oie.enroll.okta_verify.select.channel.ovOnThisDevice"
+        bundle="login" $1="<a class='ov-same-device-enroll-link' href='#'>$1</a>"}}}
+      `,
+  postRender: function() {
+    const appState = this.options && this.options.appState;
+    const model = this.model;
+
+    this.$('.ov-same-device-enroll-link').click(function() {
+      if (!appState || !model) {
+        return;
+      }
+
+      const sameDeviceChannel = 'samedevice';
+      const remediations = appState.get('remediations');
+      const selectEnrollmentChannelRemediation = remediations.find((remediation) => {
+        return remediation.name === RemediationForms.SELECT_ENROLLMENT_CHANNEL;
+      });
+
+      if (!selectEnrollmentChannelRemediation) {
+        return;
+      }
+      const idField = selectEnrollmentChannelRemediation.uiSchema?.find((schema) =>
+        schema.name === 'authenticator.id');
+      if (!idField) {
+        return;
+      }
+      // filter selected channel
+      const sameDeviceChannelField = selectEnrollmentChannelRemediation.uiSchema?.find((schema) =>
+        schema.name === 'authenticator.channel');
+      if (!sameDeviceChannelField) {
+        return;
+      }
+      sameDeviceChannelField.options = sameDeviceChannelField.options?.filter((option) =>
+        option.value === sameDeviceChannel);
```

Comment:
Same concern here about the filtered result in the event that `samedevice` is the only channel. Will that cause issues?

Reply:
I dont think it should be an issue because the user should not have access to the channel selection screen; when `samedevice` is the only channel, that means OV security is set to HIGH.  The `Or set up Okta Verify on a mobile device` link to enroll in other channels will not display when security is set to HIGH